plugins/network-elements/nuage-vsp/pom.xml

```diff
@@ -28,6 +28,13 @@
     <version>4.6.0-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>${cs.lang3.version}</version>
+    </dependency>
+  </dependencies>
```

Comment:
Is ``commons-lang3`` required by the nuage client jar?  If not, please use the equivalent facilities in Guava as that is the project's standard, common utility library.